wordproof-timestamp/includes/MetaBox.php

```diff
@@ -17,7 +17,7 @@ public function __construct() {
     public function addMetaBox() {
         add_meta_box(
                 'wordproof-meta-box',
-                'WordProof',
+                __( 'WordProof', 'wordproof-timestamp' ),
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?